### PR TITLE
Created community directory and README. Added initial profile page.

### PR DESCRIPTION
**Pull request by leighphan**
To start a member directory for the FullstackLA website, we've created a members directory for attendees to create profile pages. Adding oneself to the directory can also be an exercise in submitting pull requests.

### CODE REVIEW

**Review by machikoyasuda — CHANGES_REQUESTED** [review]
I love this start! I especially love that you added instructions. 

Here are my wording & grammar thoughts:
- Replace `members` with `community` - `community` feels more open-ended, welcoming and inclusive of anyone that comes by, even if they can only come once. `members` makes it sound a little exclusive... like you have to be regularly-attending and get a membership card or something. You can move this to the `/community/` folder via github.com interface by deleting the filename till it reaches the previous directory: 
  screenshot 2016-10-19 17 33 31
- I unofficially officially adopted the capitalization rule of don't capitalize anything unless it's the first word and a proper noun. I stole this idea from @jdorfman's awesome list: https://github.com/jdorfman/awesome-json-datasets/blob/master/CONTRIBUTING.md
- **Naming conventions**: We should add a note on the naming convention of the file name. Since we want them to be unique, I'm thinking, Github handle would be a good name for the file. What do you think?

Here are my code thoughts:
- Add real links in Markdown with this format: 

```
[Website](http://mywebsite.com)
```

Here are my thoughts on content:
- Sounds obvious but perhaps a link to GitHub profile and handle would be good too. From a GitHub profile, you can see what languages they have worked on.
- Even more things we can add eventually: preferred pronouns, answers to silly fun stand-up-esque questions like 

```
At this meetup, I am super excited to:
I used to:
The latest cool thing I learned was:
My favorite tech tip is:
```

File: members/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# FullstackLA Members
+
+In this members directory, FullstackLA members will create profiles in the form of .md files.
+
+## How to Add Your FullstackLA Member Profile
```
Comment (machikoyasuda) [inline members/README.md]: I wanna add a blurb like this, that explains our why we want to do this, and why this is important to us. Here is my off-the-top-of-my-head ramble:

The most important part about Fullstack ⚡️ is creating and building our community here. Every week at our meetups, we create a space and time to meet and pair together to keep this community growing 🌱 . We dedicate this space in this directory to give people a chance to introduce themselves and also celebrate how much our community has grown:

Comment (leighphan) [reply members/README.md]: Nice, I like this warm and welcoming intro. Added it to our README.

**Comment by machikoyasuda** [comment]
More good questions-to-fill-out-ideas:

```
I want to learn how to:
I can teach you how to:
```

I always love to write `I can teach you how to fix your bike's flat tire.`. I stole this idea from CreativeMornings' name badges at their events.

**Comment by leighphan** [comment]
Great suggestions! Added to and revised the README and profile example.

File: members/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# FullstackLA community
```
Comment (machikoyasuda) [inline members/README.md]: I think this could just be `# Community` - to be more open!

File: members/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+In this community directory, members of the FullstackLA community will create profiles in the form of .md files.
```
Comment (machikoyasuda) [inline members/README.md]: You can take out `LA` :) Cos we are not just in LA!

**Pull request merged**